packages/arb-bridge-peripherals/contracts/tokenbridge/libraries/aeWETH.sol

```diff
@@ -24,13 +24,13 @@ import "./IWETH9.sol";
 /// @title Arbitrum extended WETH
 contract aeWETH is L2GatewayToken, IWETH9 {
     function initialize(
-        string memory _name,
-        string memory _symbol,
-        uint8 _decimals,
-        address _l2Gateway,
-        address _l1Address
+        string memory name_,
```

Comment:
Any reason to place the underscore at the end rather than the start?

Reply:
https://github.com/OffchainLabs/arbitrum/pull/1371/files#r668794284